src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultItems.props

```diff
@@ -34,11 +42,11 @@
   </ItemGroup>
 
   <!-- Automatically supply project capabilities for IDE use -->
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetPlatformIdentifier)' == 'Android' ">
     <ProjectCapability Include="Mobile" />
     <ProjectCapability Include="Android" />
     <ProjectCapability Include="AndroidApplication" Condition="$(AndroidApplication)" />
-    
+
     <!-- Conflicts with our targets generator in VS+CPS: See https://work.azdo.io/1112733 -->
     <ProjectCapability Remove="LaunchProfiles" />
   </ItemGroup>
```

Comment:
Is there a reason the ProjectCapability items need to be defined in the .props?  Could they just be defined in a .targets file?

Reply:
I'll move these into a targets file.